tests/functional/modules/test_zos_blockinfile_func.py

```diff
@@ -1238,7 +1248,7 @@ def test_uss_block_insertafter_eof_with_backup_name(ansible_zos_module):
             for result in results.contacted.values():
                 assert result.get("stdout") == TEST_ENV["TEST_CONT"]
     finally:
-        ansible_zos_module.all.file(path=USS_BACKUP_FILE, state="absent")
+        ansible_zos_module.all.file(path=USS_BACKUP_FILE, state="absent") 
```

Comment:
looks like there's a trailing space character here, PEP8 might ding you for that